.github/files/build-all-projects.sh

```diff
@@ -178,6 +178,32 @@ for SLUG in "${SLUGS[@]}"; do
 	# Copy SECURITY.md
 	cp "$BASE/SECURITY.md" "$BUILD_DIR/SECURITY.md"
 
+	if [[ "$SLUG" == "plugins/jetpack" ]]; then
+		echo "Copying Jetpack files for backward compatibility."
+
+		OLD_VENDOR_DIR="$BUILD_DIR/vendor"
+		NEW_VENDOR_DIR="$BUILD_DIR/jetpack_vendor"
+		FILES_TO_COPY=(
+			"automattic/jetpack-roles/src/class-roles.php",
+			"automattic/jetpack-backup/src/class-package-version.php",
+			"automattic/jetpack-sync/src/class-package-version.php",
+			"automattic/jetpack-connection/src/class-package-version.php",
+			"automattic/jetpack-connection/src/class-urls.php",
+			"automattic/jetpack-sync/src/class-queue-buffer.php",
+			"automattic/jetpack-sync/src/class-utils.php",
+			"automattic/jetpack-connection/legacy/class-jetpack-ixr-client.php",
+			"automattic/jetpack-connection/src/class-client.php",
+			"automattic/jetpack-connection/legacy/class-jetpack-signature.php"
+		)
+
+		for file in "${FILES_TO_COPY[@]}"; do
+			if [[ -f "$file" ]]; then
+				cp "$NEW_VENDOR_DIR/$file" "$OLD_VENDOR_DIR/$file"
```

Comment:
That won't work as is, since the subdirectories do not exist. I will have to create those directories before to copy.

Reply:
Trying with `install` to see if that helps.

Reply:
`install` didn't work. @anomiex I'd be curious to know how you'd approach this. Do we have to break down the strings and `mkdir` each dir individually?